news/6858.feature

```diff
@@ -0,0 +1 @@
+This change will report any unfound packages in the pip show command output. 
```

Comment:
```suggestion
Warn about packages not found by pip show.
```

Reply:
Better thank you - I have done this as a part of the commit to also fix 2 lint errors in the tests.